ui/components/Revealer.gd

```diff
@@ -142,6 +141,10 @@ func set_text_color(value: Color) -> void:
 		_button.add_color_override("font_color", text_color)
 
 
+func _animate_height(property: String, from: float) -> void:
+	_tween.interpolate_property(self, property, from, _height, ANIM_DURATION)
```

Comment:
I´d rather not have functions with only one instruction. Here, it mostly requires you to jump somewhere else in the file to understand how the code works - you're not reusing enough code to justify the indirection. I'll change this and merge shortly.